README.md

```diff
@@ -1,20 +1,18 @@
-# <a href="http://liskhq.github.io/lisk-js/">Lisk-JS</a>
+# <a href="http://liskhq.github.io/lisk-js/">Lisk-JS</a> (soon to be renamed Lisk-Elements)
 
 Lisk JS is a JavaScript library for [Lisk - the cryptocurrency and blockchain application platform](https://github.com/LiskHQ/lisk). It allows developers to create offline transactions and broadcast them onto the network. It also allows developers to interact with the core Lisk API, for retrieval of collections and single records of data located on the Lisk blockchain. Its main benefit is that it does not require a locally installed Lisk node, and instead utilizes the existing nodes on the network. It can be used from the client as a [browserify](http://browserify.org/) compiled module, or on the server as a standard Node.js module.
 
 [![Build Status](https://jenkins.lisk.io/buildStatus/icon?job=lisk-js/development)](https://jenkins.lisk.io/job/lisk-js/job/development/)
 [![Coverage Status](https://coveralls.io/repos/github/LiskHQ/lisk-js/badge.svg?branch=development)](https://coveralls.io/github/LiskHQ/lisk-js?branch=development)
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
-[![GitHub release](https://img.shields.io/badge/version-0.4.5-blue.svg)](#)
+[![GitHub release](https://img.shields.io/badge/version-1.0.0-blue.svg)](#)
 
 ## Browser
 
 ```html
 <script src="./lisk-js.js"></script>
 <script>
-	lisk.api().searchDelegateByUsername('oliver', function (response) {
-		console.log(response);
-	});
+	lisk.APIClient({}).get.delegates({ username: 'oliver' }).then(console.log);
```

Comment:
I think the usage needs to be updated
`new lisk.APIClient(['http://localhost:4000']).delegates.get({ username: 'oliver' }).then(console.log);`